scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -89,10 +99,16 @@ abstract class H2OMOJOModel
 
   def getTrainingMetrics(): Map[String, Double] = $(trainingMetrics)
 
+  def getTrainingMetricsObject(): H2OMetrics = $(trainingMetricsObject)
+
   def getValidationMetrics(): Map[String, Double] = $(validationMetrics)
 
+  def getValidationMetricsObject(): H2OMetrics = $(validationMetricsObject)
+
   def getCrossValidationMetrics(): Map[String, Double] = $(crossValidationMetrics)
 
+  def getCrossValidationMetricsObject(): H2OMetrics = $(crossValidationMetricsObject)
+
```

Comment:
Unrelated to this PR - How do we get the Metrics CV summary? is that exposed?

Reply:
it's exposed by a [different PR](https://github.com/h2oai/sparkling-water/pull/2559), which hasn't been merged yet.

Reply:
thank you - makes sense